test/gen_dune_rules.ml

```diff
@@ -0,0 +1,30 @@
+let is_testcase name =
+  Fpath.(has_ext "dot" (v name))
+
+let pp_targets ppf filenames =
+  Fmt.pf ppf
+    {|(rule
+ (targets %a)
+ (action (run ./test.exe)))
+
+|}
+    Fmt.(list ~sep:(const string "\n          ") string) filenames
```

Comment:
Nit: you can use `@[<v>%a@]` in the format string and let fmt add the indentation (just use `Fmt.cut` as the separator).

Reply:
This change breaks left-alignment with the first list item:
```
 (rule
  (targets option-none.1.dot
           ...
-          v5n.1.dot
-          v5n.2.dot
-          v5n.3.dot)
+                option-none.2.dot
+                option-some.1.dot
+                option-some.2.dot
```
Currently trying to work out why...

Reply:
Probably due to hard-coding some `\n`s elsewhere - Fmt doesn't like it when you add your own newlines without telling it.